client/packages/invoices/src/Returns/modals/InboundReturn/ReturnQuantitiesTable.tsx

```diff
@@ -107,8 +113,8 @@ export const QuantityReturnedTableComponent = ({
 export const QuantityReturnedTable = React.memo(QuantityReturnedTableComponent);
 
 // Input cells can't be defined inline, otherwise they lose focus on re-render
-// eslint-disable-next-line new-cap
 const PackUnitEntryCell =
+  // eslint-disable-next-line new-cap
```

Comment:
yeah, this bothers me too. the `PackVariantEntryCell` isn't a component and should not be capitalised.
Needs to be renamed to be a getter fn, which it is, rather than disabling hints.

Reply:
Yeah agree - thought it was a bit much of a side quest for this PR...?

Reply:
yes, you're right. just venting uhelpfully 😉